awspricing/offers.py

```diff
@@ -468,7 +468,7 @@ def reserved_hourly(self,
 
         price_dimensions = term['priceDimensions'].values()
         hourly_dimension = next(d for d in price_dimensions
-                                if d['unit'] == 'hrs')
+                                if d['unit'] == 'Hrs')
```

Comment:
Would you mind updating this to compare via `d['unit'].lower()` to guard against a similar API change in future?